content/rs/references/rest-api/requests/bdbs/actions/backup_reset_status.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: Backup reset status database action requests
+linkTitle: backup_reset_status
+description: Reset database backup status requests
+weight: $weight
+alwaysopen: false
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases: /rs/references/rest-api/bdbs/actions/backup_reset_status
+         /rs/references/rest-api/bdbs/actions/backup_reset_status.md
+         /rs/references/restapi/bdbs/actions/backup_reset_status
+         /rs/references/restapi/bdbs/actions/backup_reset_status.md
+         /rs/references/rest_api/bdbs/actions/backup_reset_status
+         /rs/references/rest_api/bdbs/actions/backup_reset_status.md
+---
+
+| Method | Path | Description |
+|--------|------|-------------|
+| [PUT](#put-bdbs-actions-backup-reset-status) | `/v1/bdbs/{uid}/actions/backup_reset_status` | Reset database backup status |
+
+## Reset database backup status {#put-bdbs-actions-backup-reset-status}
+
+	PUT /v1/bdbs/{int: uid}/actions/backup_reset_status
+
+Reset the database's `backup_status` to idle if a backup is not in progress and clears the value of the `backup_failure_reason` field.
```

Comment:
```suggestion
Resets the database's `backup_status` to idle if a backup is not in progress and clears the value of the `backup_failure_reason` field.
```

Reply:
Not sure if "Reset' needs an "s" or if "clears" need to delete an "s". They should be consistent.

Reply:
Fixed.